doc/classes/VisualShaderNodeCompare.xml

```diff
@@ -1,45 +1,62 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualShaderNodeCompare" inherits="VisualShaderNode" category="Core" version="3.2">
 	<brief_description>
+		A comparsion function for common types within visual shader graph.
 	</brief_description>
 	<description>
+		Compares [code]a[/code] and [code]b[/code] of [member type] by [member function]. Returns a boolean scalar. Translates to "if" instruction in shader code.
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
 	</methods>
 	<members>
 		<member name="condition" type="int" setter="set_condition" getter="get_condition" enum="VisualShaderNodeCompare.Condition" default="0">
+			Extra condition which applied if [member type] setted to [constant CTYPE_VECTOR].
 		</member>
 		<member name="function" type="int" setter="set_function" getter="get_function" enum="VisualShaderNodeCompare.Function" default="0">
+			A comparsion function.
 		</member>
 		<member name="type" type="int" setter="set_comparsion_type" getter="get_comparsion_type" enum="VisualShaderNodeCompare.ComparsionType" default="0">
+			A type of comparsion parameters.
```

Comment:
The type to be used in the comparison. See [enum ComparsionType] for options.

**Note:** It seems that "comparison" is wrongly spelled in the code as well. We will have to change "Comparsion" to "comparison" in the code as well

Reply:
Yeah, in 4.0.